.github/workflows/bf-review.yml

```diff
@@ -75,7 +80,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        service: ["bf-bknd", "bf-appl"]
+        service: ["bf-bknd", "bf-appl", "bf-hand"]
```

Comment:
the name must be `bf-hdlr` because that is the reserved namespace name, otherwise it does not work

Reply:
Same applies access to Vault that is based on the same